tensorrt_llm/_torch/pyexecutor/model_engine.py

```
@@ -863,7 +906,8 @@ def _round_up_batch_size(self, batch_size: int) -> int:
     def _maybe_get_cuda_graph(
         self,
         batch: ScheduledRequests,
-        spec_config: Optional["DecodingBaseConfig"] = None
+        spec_config: Optional["DecodingBaseConfig"] = None,
```

Ruff (0.12.2)

Undefined name `DecodingBaseConfig`

(F821)